kibana-reports/package.json

```diff
@@ -1,49 +1,37 @@
 {
   "name": "opendistro_kibana_reports",
-  "version": "1.10.1.0",
+  "version": "1.12.0.0",
   "description": "Kibana Reporting Plugin",
   "license": "Apache-2.0",
-  "main": "index.js",
+  "main": "index.ts",
   "kibana": {
-    "version": "7.9.1",
+    "version": "7.10.0",
     "templateVersion": "1.0.0"
   },
   "scripts": {
-    "preinstall": "node ../../preinstall_check",
     "kbn": "node ../../scripts/kbn",
     "es": "node ../../scripts/es",
     "lint": "eslint .",
-    "start": "plugin-helpers start",
-    "test:server": "plugin-helpers test:server",
-    "test:browser": "plugin-helpers test:browser",
-    "build": "plugin-helpers build",
-    "test:jest": "../../node_modules/.bin/jest --config ./test/jest.config.js",
-    "test": "jest --config ./test/jest.config.js",
-    "cypress:run": "./node_modules/.bin/cypress run",
-    "cypress:open": "./node_modules/.bin/cypress open"
+    "start": "yarn plugin_helpers start",
+    "test:server": "yarn plugin_helpers test:server",
+    "test:browser": "yarn plugin_helpers test:browser",
```

Comment:
I guess these `start/test:server/test:browser`are not supported by plugin-helper anymore. They are supported in only [old version](https://github.com/elastic/kibana/tree/v7.4.1/packages/kbn-plugin-helpers) now. You may remove them if not used.

Reply:
They are not supported, but I'm looking forward to get a alternative way for the original function.

Reply:
good catch, will do